### DISCUSSION TITLE
Can schema description be included in any ZodError?

### DISCUSSION THREAD

**Discussion by dmeehan1968** [colinhacks/zod#2594] [Q&A] [answered]
I may be missing something, but can't see a way to do the following:

I have schemas, but when a ZodError occurs, the issues only shows the path (if any) to where the validation failed, but doesn't give a context, other than the stack which is not useful in my environment as it gets transpiled and loses its source locations (yes, I could add source map support, but don't have another need to do so).

Each schema has a `describe()` method and the `description` getter, but even in a custom errorMap these are not available, as the issue seems not to provide access to the schema.  Whilst I could write custom errorMap's for each schema, this would be simple to include via a global `setErrorMap` if only the schema/description was made availiable.

Thoughts?

**Comment by rtorrente** [comment]
+1 for this feature too

Have you managed to get the schema that trigger the ZodError ?

**Comment by JacobWeisenburger** [comment, marked as answer]
This is probably better suited to be an issue as a new feature request. Because a discussion is more if you are looking for a workaround and not caring about getting a new feature.
